src/FetchWorkerTool.worker.js

```diff
@@ -55,7 +55,7 @@ const onMessage = ({data: job}) => {
             return Promise.reject(result.status);
         })
         .then(buffer => complete.push({id: job.id, buffer}))
-        .catch(error => complete.push({id: job.id, error}))
+        .catch(error => complete.push({id: job.id, error: error.message || `Failed request: ${job.url}`}))
         .then(() => jobsActive--);
```

Comment:
Should this be a `.finally`?
```suggestion
        .finally(() => jobsActive--);
```

Reply:
I experimented with this and it doesn't seem to help. We should investigate further but for now I'll leave this off.